Cdnfsd_TransparentCDN_Page_View.js

```diff
@@ -0,0 +1,70 @@
+
+jQuery(function($) {
+	function w3tc_stackpath_resize(o) {
+		o.options.height = jQuery('.w3tc_cdn_stackpath2_fsd_form').height();
```

Comment:
This file has many references to "stackpath2".  Please update all files in this pull request to reference "transparent", to avoid conflicts.